jekyll/_cci2/api-developers-guide.md

```diff
@@ -4,12 +4,12 @@ title: "CircleCI API Developer's Guide"
 short-title: "Developer's Guide"
 description: "API guide for internal and external CircleCI developers"
 categories: [getting-started]
-contentTags: 
+contentTags:
   platform:
   - Cloud
 ---
 
-This API Developer's Guide was written to assist developers in quickly and easily making API calls to CircleCI services to return detailed information about users, pipelines, projects and workflows. The API v2 Specification itself may be viewed in the [Reference documentation]({{site.baseurl}}/api/v2).
+This API Developer's Guide was written to assist developers in quickly and easily making API calls to CircleCI services to return detailed information about users, pipelines, projects and workflows. The API v2 Specification itself may be viewed in the [Reference documentation](https://circleci.com/docs/api/v2/).
```

Comment:
using full URL so it is rendered as an external link and HTMLProofer can stop complaining about it

Reply:
Ah I see!